<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FamilySearchDiag</class>
 <widget class="QDialog" name="FamilySearchDiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="searchTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>761</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lname">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="searchLabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Last Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>570</y>
     <width>254</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="newFamily">
      <property name="text">
       <string>New Family</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="add">
      <property name="text">
       <string>Add</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="tableGroupBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>120</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Table</string>
   </property>
   <widget class="QRadioButton" name="playerTableRadio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Player Table</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="familyTableRadio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Family Table</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>playerTableRadio</tabstop>
  <tabstop>familyTableRadio</tabstop>
  <tabstop>lname</tabstop>
  <tabstop>searchTable</tabstop>
  <tabstop>newFamily</tabstop>
  <tabstop>cancel</tabstop>
  <tabstop>add</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
